### PR TITLE
Extract debug info and module output properties from Driver.swift

### PR DESCRIPTION
**Pull request by sjavora**
I noticed that there were groups of properties in the driver that were returned as tuples and then assigned. It seemed natural to group these in structs.

### CODE REVIEW

File: Sources/SwiftDriver/Driver/ModuleOutputInfo.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// How should the Swift module output be handled?
+public enum ModuleOutput: Equatable {
```
Comment (sjavora) [inline Sources/SwiftDriver/Driver/ModuleOutputInfo.swift:14]: Should this be nested in `ModuleOutputInfo`?

**Comment by sjavora** [comment]
Thanks, fixed.

File: Sources/SwiftDriver/Driver/ModuleOutputInfo.swift (hunk 1 above)
Comment (artemcm) [reply Sources/SwiftDriver/Driver/ModuleOutputInfo.swift:14]: Yeah, I think that would make sense.

Comment (sjavora) [reply Sources/SwiftDriver/Driver/ModuleOutputInfo.swift]: Done.

**Comment by owenv** [comment]
@swift-ci test

**Comment by owenv** [comment]
@swift-ci test

**Comment by owenv** [comment]
🚢

**Pull request merged**